frontends/web/src/routes/account/info/info.tsx

```diff
@@ -80,16 +81,27 @@ export const Info = ({
                   </button>
                 </p>
               ) : null}
-              <SigningConfiguration
-                key={viewXPub}
-                account={account}
-                code={code}
-                info={config}
-                signingConfigIndex={viewXPub}>
-                <BackButton enableEsc>
-                  {t('button.back')}
-                </BackButton>
-              </SigningConfiguration>
+              { isBitcoinOnly(account.coinCode) && (viewXPub === 1) ? (
```

Comment:
checking the index is fragile. you can check info.signingConfigurations[viewXPub] directly:  `signingConfig.bitcoinSimple && signingConfig.bitcoinSimple.scriptType === 'p2tr'` or something like that